sycl/CMakeLists.txt

```
@@ -223,17 +223,21 @@ endif()
 # This function allows building multiple libraries with the same options.
 # Currently used by sycl and plugins library.
 # Currently handles linking with libcxx support and gcc workaround
-function( add_common_options LIB_NAME)
+function( add_common_options)
```

I don't think `add_common_options` functions should allow empty argument list.
The code below doesn't check that `ARGV0` is provided, so it might fail on wrong usages of the function.
I suggest keep `LIB_NAME` named argument.
```suggestion
function( add_common_options LIB_NAME)
```